types/staking.go

```diff
@@ -119,6 +119,9 @@ type ValidatorSet interface {
 	// Delegation allows for getting a particular delegation for a given validator
 	// and delegator outside the scope of the staking module.
 	Delegation(Context, AccAddress, ValAddress) Delegation
+
+	// MaxValidators returns the maximum amount of bonded validators
+	MaxValidators(Context) uint16
```

Comment:
this is fine for now, but ultimately this whole interface should be migrated into expected keepers

Reply:
Agreed. Separate PR 👍